service/src/main/kotlin/no/nav/su/se/bakover/service/ServiceBuilder.kt

```diff
@@ -42,6 +44,9 @@ class ServiceBuilder(
             dokArkiv = clients.dokArkiv,
             dokDistFordeling = clients.dokDistFordeling
         )
+        val oppgaveService = OppgaveServiceImpl(
```

Comment:
Kunne byttet ut de andre konstruktørene til servicene og bruke OppgaveService istedet for OppgaveClient

Reply:
👍